api/src/main/java/org/openmrs/module/appointments/service/impl/AppointmentServiceDefinitionServiceImpl.java

```diff
@@ -47,6 +48,11 @@ public AppointmentServiceDefinition getAppointmentServiceByUuid(String uuid) {
         return appointmentServiceDao.getAppointmentServiceByUuid(uuid);
     }
 
+    @Override
+    public List<AppointmentServiceDefinition> search(AppointmentServiceSearchRequest searchRequest) {
+        return appointmentServiceDao.search(searchRequest);
```

Comment:
here you can separate the web payload from the service layer search object.
something like - _searchRequest.toSeachParams()_